README.md

```diff
@@ -42,6 +42,17 @@ This section describes the workflow we are going to follow when working in a new
 
 Forking the project will create a copy of that project in your own GitHub account, you will commit your work against your own repository.
 
+#### Quick start
+
+1. Navigate to the project folder.
+2. Run `yarn install`
+3. Run `yarn test`
+4. Type `a` to run all tests
+5. Type `q` to quit (this will start cypress and set that up)
+6. Navigate to [http://localhost:3000](http://localhost:3000) to see the site
+
+For other options, please see "Available Scripts" below.
```

Comment:
```suggestion
For other options, please see [Available Scripts](https://github.com/Coding-Coach/find-a-mentor#available-scripts) below.
```
This could be a link so its easier to go to Available Scripts Section

Reply:
I think that steps 3 to 5 should be replaced for `yarn start`. 
The server will shutdown after Cypress finish running the tests if we run the app with `yarn test`

Reply:
`yarn start` didn't work for me because the react-scripts weren't installed, so we need a step before that.